contracts/script/deploy/verifiable/PrintMultisigCalldata.s.sol

```diff
@@ -36,7 +34,6 @@ contract PrintMultisigCalldata is Script {
 
     /// @dev override this if you don't want to use the environment to get the config path
     function _initConfig() internal virtual {
-        deploymentInitializer = vm.envAddress("DEPLOYMENT_INITIALIZER");
```

Comment:
was this just not used? removing this doesn't seem related to PR title/description.

Reply:
Yes, It was just something I caught while I was working on the PR. I can separate it out, but I think this PR is still acceptably small.